src/pages/singlevideo/SingleVideo.jsx

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from 'react'
+import { Navbar, Footer, VideoCard } from 'components'
+import { useParams, useNavigate, Link } from 'react-router-dom'
+import { videos } from 'backend/db/videos'
+export function SingleVideo() {
+  let { videoId } = useParams()
+  const video = videos.find(product => product._id === videoId)
+  const {
+    _id,
+    title,
+    description,
+    views,
+    creator,
+    duration,
+    thumbnail,
+    alt,
+    category,
+    avatar,
+    link,
+  } = video
+  return (
+    <div className="home__container">
+      <Navbar />
+      <div className="main__container single-video-container">
+        <div className="single-video text__lg">
+          <div className="video-player">
+            <iframe
+              className="video-iframe"
+              src={link}
+              title={title}
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowFullScreen></iframe>
+          </div>
+          <div className="single-video-footer">
+            <div className="text__xl font__bold">{title}</div>
+            <div className="single-video-details">
+              <div className="single-video-icons">
+                <span className="pointer">
+                  <i className="fa-regular fa-thumbs-up"></i> Like
+                </span>
+                <span className="pointer">
+                  <i className="fa-regular fa-thumbs-down"></i>Dislike
+                </span>
+                <span className="pointer">
+                  <i className="fa-regular fa-heart"></i>Add to Watch Later
+                </span>
+                <span className="pointer">
+                  <i className="fa-solid fa-layer-group"></i>Add to Playist
```

Comment:
Just a suggestion: Perhaps you can create a list of single video icons and map the array which would prevent the repetition of code.

Reply:
sure will do it